xlml/apis/task.py

```diff
@@ -176,7 +176,8 @@ def run(
 
     Attributes:
       gcs_location: GCS path for all artifacts of the test.
-      use_vertex_tensorboard: Set to True to view workload data on Vertex AI Tensorboard.
+      use_vertex_tensorboard: Set to True to view workload data on
```

Comment:
Nit: what's this change? Formatting?

Reply:
Yep, started to get line-too-long errors